CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
     - top lists ordered by point values of players
     - `npcs` list now supports global variables as NPC ID in NPC Holograms
 - menus now support global variables
+- counting-objectives now support variables
```

Comment:
This will not be understood by a user, they don't know what a counting objective is.
```suggestion
- Some objectives now support variables for their amount options:
  - BlockObjective
  - BreedObjective
  - BrewObjective
  - ConsumeObjective
  - CraftingObjective
  - EnchantObjective
  - EntityInteractObjective
  - FishObjective
  - JumpObjective
  - KillPlayerObjective
  - MobKillObjective
  - PickupObjective
  - ShearObjective
  - SmeltingObjective
  - TameObjective
```

Reply:
Every other objective (except the ObjectivePaymentEvent from the Jobsreborn Integration) already supports Variable Numbers, so I suggest to change it to "all objectives now support variables as amount", when I changed the named Objecitve.